infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -1336,15 +1366,18 @@ private void deleteFilesMatchingFilter(FileFilter fileFilter) {
     }
 
     /**
+     * TODO(Maithem) remove this method. Obtaining a new instance should happen
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)